cfpb/cfgov-refresh
# Line length feature flag

This is an attempt to move some CSS behind a Wagtail feature flag. See https://[GHE]/CFGOV/platform/issues/237 for line length background.

## Additions

- Feature flag gulp task.
- Feature flag CSS directory to put relevant stylesheets in.
- Feature flag conditional at the bottom of `base.html`.

## Testing

- Pull down branch.
- `gulp styles`
- Log into the wagtail admin and add a `NEW_LINE_LENGTH` feature flag.

![screen shot 2016-08-30 at 4 28 53 pm](https://cloud.githubusercontent.com/assets/1060248/18105840/fdc89844-6ece-11e6-956c-3fedfb2852e1.png)

When it's enabled (most) lines should wrap after `41.875rem`. For example, try visiting http://localhost:8000/office-civil-rights/eeo-policy-and-reports/ before and after toggling the `NEW_LINE_LENGTH` checkbox.

## Notes
- After the feature has been reviewed by stakeholders we'll remove this flag and merge the CSS into the appropriate stylesheet.

## Review

- @rosskarchner 
- @virginiacc 
- @sebworks 

## Checklist
